spec/services/exports/export_transfers_csv_service_spec.rb

```diff
@@ -100,5 +100,84 @@
         expect(header_row).to eq(expected_headers)
       end
     end
+
+    context "when an organization's item exists but isn't in any transfer" do
+      let(:unused_item) { create(:item, name: "Unused Item", organization: organization) }
+      let(:generated_csv_data) do
+        # Force unused_item to be created first
+        unused_item
```

Comment:
Would it be simpler to replace `let` with `let!` on line 105, and then you don't have to do this? Same below.